code/game/machinery/tcomms/nttc.dm

```diff
@@ -27,11 +27,16 @@
 		// Assistant
 		"Assistant" = "radio",
 		// Command (Solo command, not department heads)
+		"Acting Captain" = "comradio",
+		"Acting Head of Personnel" = "comradio", //extremely unlikely to ever exist, yet I had it in my old NTTC script file
+		"Acting HoP" = "comradio", //ditto
```

Comment:
Ive already been against hardcoding this many of the things, given how different `Acting` titles can be. It can be any of:

- Acting Head of Personnel
- Acting HoP
- Acting HOP
- HOP (Acting)

Etc etc etc, and it sets a really slippery slope for even more exceptions to be thrown in

Reply:
Two spellings per title isn't too heavy of a requirement, especially when one of them is literally just the old title with "Acting" in front of it.
Plus if they want to give the Acting Chief Engineer the job title of "ACE" that's on them, and it's not like this PR breaks it, or prevents it from being completely functional.

I understand that NTSL had to be removed for security reasons, but there are some good quality of life things that were lost that this PR is trying to restore.

Reply:
Plus, of all the acting head jobs that do have an "of" in them, the only one that comes to mind which is seen commonly enough is the HoS.

Acting HoPs are most of the time non-existent, since if there is a Captain or Acting Captain around, they do the HoP job as well, so the idea of having "acting head Of personnel" as job title is pretty much negligible.

That does leave only the alternate spellings of Acting Head of Security as "more exceptions" that have to be considered, but the comfort of having the voice of the current lead of your apartment clearly distinguishable from the rest still outweighs the risk of alternative spellings.